README.md

```diff
@@ -41,128 +41,6 @@ Angular DevKit.
 DevKit's goal is to provide a large set of libraries that can be used to manage, develop, deploy and
 analyze your code.
 
-# Getting Started - Local Development
```

Comment:
We should at least link to the new `DEVELOPER.md` doc from the `README`, maybe somewhere under Contributing. That way someone who wants to make a PR can still find the doc, but people who just want to use the CLI will recognize that it isn't for them.